apis/kubekey/v1alpha1/kubernetes_types.go

```diff
@@ -27,6 +27,7 @@ type Kubernetes struct {
 	NodeCidrMaskSize         int                  `yaml:"nodeCidrMaskSize" json:"nodeCidrMaskSize,omitempty"`
 	ApiserverCertExtraSans   []string             `yaml:"apiserverCertExtraSans" json:"apiserverCertExtraSans,omitempty"`
 	ProxyMode                string               `yaml:"proxyMode" json:"proxyMode,omitempty"`
+	Nodelocaldns             bool                 `yaml:"nodelocaldns" json:"nodelocaldns,omitempty"`
```

Comment:
`Nodelocaldns` the default value is `false`. I'm not sure it is the same as before. Maybe it missing a default value configure function in `default.go`.